python/hsfs/feature_store.py

```diff
@@ -915,6 +1004,11 @@ def get_transformation_function(
     def get_transformation_functions(self):
         """Get  all transformation functions metadata objects.
 
+        !!! example
+            ```python
+            func_list = [t_func.name for t_func in fs.get_transformation_functions()]
+            ```
```

Comment:
This snippet just lists names of transformation functions. I this this will be enough `func_list =  fs.get_transformation_functions()`